### DISCUSSION TITLE
Može li više faktura da se kreira iz jednog ugovora?

### DISCUSSION THREAD

**Discussion by Bmiljan** [NPSBeograd/NPS-Support#81] [Q&A] [answered]
Može li više faktura da se kreira iz jednog ugovora?

**Comment by IvanaBasara** [comment, marked as answer]
Ne postoji mogućnost da se iz jednog ugovora odjednom kreiraju sve fakture koje su predviđene za konkretan ugovor. Na primeru sa obuke je na ugovoru definisan period servisiranja 12 meseci, tako da se faktura kreira na mesečnom nivou i ne može druga faktura da se kreira dok ne istekne vremenski period tekuće fakture.
